data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life signs aboard, along with numerous other minor lifesigns, perhaps the trees that you can see in the massive dome atop the craft?`
```

Comment:
```suggestion
			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life signs aboard, along with numerous other minor life-signs, perhaps the tree-like things that you can see in the massive dome atop the craft?`
```

The player doesn't know they are trees, just something that looks like trees. Not set on this change, just worth considering.

Reply:
tree sized plant like?

Reply:
Yeah. Hence food for thought, not a "required change."

It's an alien life form that resembles a tree... Or for all we know just a big green piece of decor. lol. 

I'm also fine with it staying as just calling them "trees" since that's what they look like.

Reply:
tree-like plants/plantlife?

Reply:
I would say leave it as trees, since the most obvious first thought most would have is just tree, our minds don't often go to "Well, it MIGHT be a replica or some alien tree" especially since the game hasn't had the player experience any alien like trees yet as far as I know, most planets just have mostly normal trees from the images. (which while misleading, is all the player sees outside the description on planets)

life-signs is good though, sounds nicer